bgpd/bgp_zebra.c

```diff
@@ -1010,6 +1010,9 @@ void bgp_zebra_announce(struct bgp_node *rn, struct prefix *p,
 	/* Metric is currently based on the best-path only */
 	metric = info->attr->med;
 	for (mpinfo = info; mpinfo; mpinfo = bgp_info_mpath_next(mpinfo)) {
+		if (valid_nh_count >= MULTIPATH_NUM)
```

Comment:
shouldn't this be `multipath_num` in bgp?

Reply:
I think it would read better, but I tried to touch the code as little as possible when I was doing the ZAPI refactoring, to facilitate the review process.

Reply:
sorry, I misunderstood you... yes, I'll fix this.